tests/fixtures/prepare_environment_fixture.py

```diff
@@ -1,31 +1,34 @@
+from __future__ import annotations
+from typing import Any
 import dataclasses
 import os
 from inspect import cleandoc
 from typing import Optional
 
 import boto3
 import pyexasol
+import exasol.bucketfs as bfs
 import pytest
 from click.testing import CliRunner
 
 from exasol_sagemaker_extension.deployment import deploy_cli
-from tests.ci_tests.utils.parameters import db_params, \
-    reg_model_setup_params, cls_model_setup_params
+from tests.ci_tests.utils.parameters import (
+    get_arg_list, reg_model_setup_params, cls_model_setup_params)
 
 
-def __open_schema(db_conn, model_setup):
+def __open_schema(db_conn: pyexasol.ExaConnection, model_setup):
```

Comment:
Why double underline?

Reply:
I don't know, it wasn't me who wrote it like that.